packages/rest-api/src/tests/bridgeRoute.test.ts

```diff
@@ -200,6 +201,7 @@ describe('Bridge Route with Real Synapse Service', () => {
       fromToken: USDC.addresses[1],
       toToken: USDC.addresses[10],
       amount: '1000',
+      originUserAddress: '0x742d35Cc6634C0532925a3b844Bc454e4438f44e',
```

Comment:
Let's also add the test where only `destAddress` is supplied to ensure that future updates don't break this invariant.

Reply:
fixed in latest commit